inky/inky.go

```diff
@@ -148,13 +193,31 @@ func New(p spi.Port, dc gpio.PinOut, reset gpio.PinOut, busy gpio.PinIn, o *Opts
 	case PHAT:
 		d.bounds = image.Rect(0, 0, 104, 212)
 		d.flipVertically = true
+	case PHAT2:
+		d.bounds = image.Rect(0, 0, 122, 250)
+		d.flipVertically = true
 	case WHAT:
 		d.bounds = image.Rect(0, 0, 400, 300)
 	}
 
 	return d, nil
 }
 
+// NewDetected tries to open a handle to an Inky pHat or wHAT automatically detected from EEPROM.
+// For a Raspberry Pi running linux the SPI port will usually be 'SPI0.0' and I2C bus '1'.
+func NewDetected(port spi.Port, bus i2c.Bus) (*Dev, error) {
```

Comment:
I think I'd like to have either DetectOpts() or NewDetected() exported but not both. I'm edging on DetectOpts since it's more composable and removes the dependency on gpioreg.

Reply:
As a user I'd personally prefer both, DetectOpts() for flexibility and NewDetected() ease of access, but I can understand why only having one of them exported would be preferable. In that case I think it would be better to keep DetectOps(), for the reasons you have mentioned as well as the ability to still detect the Inky model when using a non standard pinout on the device.